packages/roosterjs-content-model-plugins/test/autoFormat/AutoFormatPluginTest.ts

```diff
@@ -1092,7 +1092,13 @@ describe('Content Model Auto Format Plugin Test', () => {
                             segments: [
                                 {
                                     segmentType: 'Text',
-                                    text: 'test—test',
+                                    text: 'test—tes',
```

Comment:
I wonder why the result is like this. I think the result should just be a single text segment

Reply:
This test shouldn't pass, but I think it passed, because the selection marker was not in the model.